packages/pnpm-config/README.md

```diff
@@ -0,0 +1,40 @@
+# @pagopa/pnpm-config
+
+Default configuration for PagoPA DX projects using [pnpm](https://pnpm.io).
+
+## Default configuration
+
+- Sets [workspacePackagePatterns](https://pnpm.io/settings#workspacepackagepatterns) to `["apps/*", "packages/*"]`.
+- Sets [linkWorkspacePackages](https://pnpm.io/cli/run#linkworkspacepackages) to `true`.
+- Sets [packageImportMethod](https://pnpm.io/settings#hoistpattern) to `clone-or-copy`
+- Adds `dx` [catalog](https://pnpm.io/catalogs) which includes recommended version of the packages supported by the PagoPA DX toolchain. Here you can find the full [list of packages included](https://github.com/pagopa/dx/blob/main/packages/pnpm-config/pnpmfile.cjs).
+
+## Use the PagoPA DX catalog
+
+To use the PagoPA DX catalog, you can specify the `catalog:dx` version for supported packages in your `package.json`. This will ensure that you are using the recommended versions of packages that are compatible with the PagoPA DX toolchain.
+
+```jsonc
+{
+  // .....
+  "devDependencies": {
+    "eslint": "catalog:dx",
+    "vitest": "catalog:dx",
+  },
+}
+```
+
+## Installation
+
+```
+pnpm add --config @pagopa/pnpm-config
+```
+
+This will add `@pagopa/pnpm-config` to the [configDependencies](https://pnpm.io/config-dependencies) field in your `pnpm-workspace.yaml`.
+
+### Use the pnpmfile
+
+To use the `pnpmfile.cjs` provided by `@pagopa/pnpm-config`, you need to add the following line to your `pnpm-workspace.yaml`:
```

Comment:
suggestion: I'd clarify what this step enables (what if I miss this? is this mandatory?)

Reply:
I discovered that this step can be omitted using the `pnpm-plugin-*` prefix for config packages, so I renamed the workspace to `pnpm-config-pagopa`